pages/docs/reference/faq.md

```diff
@@ -108,7 +108,7 @@ There are too many companies using Kotlin to list, but some more visible compani
  
 ### Who develops Kotlin?
 
-Kotlin is primarily developed by a team of engineers at JetBrains (current team size is 90+). The lead language designer is 
+Kotlin is primarily developed by a team of engineers at JetBrains (current team size is 100+). The lead language designer is 
```

Comment:
250 contributors - isn't it bigger now?

Reply:
I believe this is close to the truth:
- 407: the exact number of  contributors (https://github.com/JetBrains/kotlin)
- 90-100: Kotlin team members
- 40-50:  former team members and interns who already left.